package.json

```diff
@@ -560,6 +561,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-storybook": "^0.6.15",
     "eta": "^3.2.0",
+    "ethers": "5.7.0",
```

Comment:
Not directly used, but expected by `@account-abstraction/contracts` during tests.
Without this, tests would fail as the transitive ethers v6+ would get used.